netket/machine.py

```diff
@@ -1,5 +1,256 @@
 from ._C_netket.machine import *
+import numpy as _np
 
 
 def MPSPeriodicDiagonal(hilbert, bond_dim, symperiod=-1):
     return MPSPeriodic(hilbert, bond_dim, diag=True, symperiod=symperiod)
+
+
+class CxxMachine(Machine):
+    def __init__(self, hilbert):
+        super(CxxMachine, self).__init__(hilbert)
+
+    def log_val(self, x):
+        r"""Returns the logarithm of the wave function at point x.
```

Comment:
Nitpick: "At point x" is a bit unclear. How about "for the visible configuration v"?